### PR TITLE
adds new coordinator metric 'segment/unavailable/count'

### PR DESCRIPTION
**Pull request by michaelschiff**
issue #3173

### CODE REVIEW

**Comment by gianm** [comment]
Thanks for the contribution! Could you please add docs to metrics.md too?

👍  other than that.

**Comment by nishantmonu51** [comment]
👍 after docs are added

File: docs/content/operations/metrics.md [hunk 1]
```diff
@@ -142,6 +142,7 @@ These metrics are for the Druid coordinator and are reset each time the coordina
 |`segment/size`|Size in bytes of available segments.|dataSource.|Varies.|
 |`segment/count`|Number of available segments.|dataSource.|< max|
 |`segment/overShadowed/count`|Number of overShadowed segments.||Varies.|
+|`segment/unavailable/count`|Number of segments left to load until segments that should be loaded in the cluster are available for queries.|datasource.|0|
```
Comment (fjy) [inline docs/content/operations/metrics.md]: can you add a note if this includes or doesn't include replication?

**Comment by michaelschiff** [comment]
This metric corresponds to `/druid/coordinator/v1/loadstatus?simple`

I'm thinking its worth it to add a metric for `/druid/coordinator/v1/loadstatus?full` as well

**Comment by gianm** [comment]
👍  LGTM, thanks!

**Pull request merged**